merge_state_block/spec.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Output": "Non state setting signals are passed through with **state** set to the attribute `state_name`.",
+    "Dependencies": [
+        "None"
+    ],
+    "Block": "MergeState",
+    "Version": "1.0.0",
+    "Description": "Maintains a **state** and merges that state (with name `state_name`) with signals that are input through the *getter* input.\nPersistence is used to maintain the **state**. If `use_persistence` is enabled, that value will populate the initial state and _not_ the `initial_state` configuration value. Any new states (from new groups) will use the configured `initial_state` for their initial state though",
+    "Commands": {
+        "current_state (parameter": " group): Gets the current state for a given group. Response is a dictionary with `state` and `group` keys. If `group` is not specified, then a list is returned with a dictionary for each group. If the specified group does not exist, then an empty dictionary is returned."
+    },
+    "Properties": {
+        "state_name": " String property that is the name of the appended state",
```

Comment:
What's up with these leading spaces? Are these needed or is this just residual from a script?